fastestimator/trace/io/best_model_saver.py

```diff
@@ -63,8 +64,12 @@ def metric(self) -> str:
         return self.inputs[0]
 
     def on_epoch_end(self, data: Data) -> None:
-        # No model will be saved when save_dir is None, which makes smoke test easier.
-        if self.save_dir and self.monitor_op(data[self.metric], self.best):
```

Comment:
why are you getting rid of this functionality?

Reply:
brought back